shipyard_blueprints/box/shipyard_box/cli/download.py

```diff
@@ -207,41 +194,53 @@ def get_client(service_account):
         client.user().get()
         return client
     except BoxOAuthException as e:
-        print(
-            f"Error accessing Box account with pervice account "
-            f"developer_token={developer_token}; client_id={client_id}; "
-            f"client_secret={client_secret}"
+        print("Error accessing Box account with service account ")
+        raise e
+
+
+def get_folder(client, folder_name):
+    """
+    Returns the folder id for the Box client if it exists.
```

Comment:
Added a function because the sdk query will return anything that matches a string with the query params. This ensures parent/subfolder actually matches the `subfolder` name. Box does not return the full paths of folders.